docs/build/isc/v1.1/docs/tutorials/defi-lend-borrow-tutorial-part-1.md

```diff
@@ -0,0 +1,86 @@
+# DeFi Lend Borrow
+
+DeFi Lend Borrow is a decentralized finance (DeFi) application that enables users to lend and borrow assets on the Shimmer EVM testnet. The project is built using Solidity and Hardhat, with the core functionality provided by smart contracts.
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Contracts Overview](#contracts-overview)
+  - [IToken](#itoken)
+  - [InterestRateModel](#interestratemodel)
+  - [ITokenManager](#itokenmanager)
+  - [MockERC20](#mockerc20)
+- [Usage](#usage)
+- [Testing](#testing)
+- [Deployment](#deployment)
+- [License](#license)
```

Comment:
No need for this, the ToC gets autogenerated by docusarurus. As well, there's broken anchors.